lib/custom-resource-handlers/pgpSecretDockerfile

```diff
@@ -0,0 +1,16 @@
+# Use a NodeJS 20.x runtime
+FROM public.ecr.aws/lambda/nodejs:20-x86_64
+
+# install openssel and
+RUN dnf install -y openssl
+ENV LD_LIBRARY_PATH=""
```

Comment:
to understand, why we have this as an empty string ?

Reply:
I faced the issue mentioned in [this thread](https://stackoverflow.com/questions/77527321/openssl-command-not-found-on-amazon-linux-2023), and so I applied the suggested solution, and it worked for me